src/nextjournal/clerk/eval.clj

```diff
@@ -148,7 +148,10 @@
           (assoc :nextjournal/interned @!interned-vars))))
     (catch Throwable t
       (let [triaged (main/ex-triage (Throwable->map t))]
-        (throw (ex-info (main/ex-str triaged) triaged))))))
+        (throw (ex-info (main/ex-str triaged)
+                        (assoc triaged
+                               :clerk/form form
+                               :clerk/file file)))))))
```

Comment:
maybe drop the `clerk` ns?

Reply:
Can we maybe reuse the `clojure.core/file` name like in https://github.com/nextjournal/clerk/blob/990749d3218b35765d8f9d8a826638e53f6d552d/src/nextjournal/clerk/analyzer.clj#L130-L132?